packages/gestalt/src/SheetMobile/PrimaryAction.js

```diff
@@ -0,0 +1,50 @@
+// @flow strict
+import { type Node, type ElementConfig } from 'react';
+import Button from '../Button.js';
+import Link from '../Link.js';
+
+type Props = {|
+  accessibilityLabel: string,
+  href?: string,
+  label: string,
+  onClick?: $ElementType<ElementConfig<typeof Button>, 'onClick'>,
+  rel?: $ElementType<ElementConfig<typeof Link>, 'rel'>,
+  size?: $ElementType<ElementConfig<typeof Button>, 'size'>,
+  target?: $ElementType<ElementConfig<typeof Link>, 'target'>,
+|};
+
+export default function ToastPrimaryAction({
+  accessibilityLabel,
+  href,
+  label,
+  onClick,
+  rel,
+  size = 'lg',
+  target,
+}: Props): Node {
+  if (href && label)
+    return (
+      <Button
+        accessibilityLabel={accessibilityLabel}
+        href={href}
+        rel={rel}
+        target={target}
+        role="link"
+        text={label}
+        size={size}
+        onClick={onClick}
+        color="red"
+      />
+    );
+
+  if (label)
```

Comment:
I don't think we need the checks for `label` since it's required and this is an internal component. The logic can be something like
```js
if (href) {
  return <Button role="link" />;
}

return <Button role="button" />;
```

Reply:
+1, I copy pasted, which would be dried in a follow up, and that was the previous code. Agreed. DONE